src/css/_hero-stats.scss

```diff
@@ -119,9 +119,6 @@ hero-stats {
 		@include absolute(left 0 top 0);
 		@include size(100%);
 		z-index: -1;
-		@include ie {
```

Comment:
this css was breaking homepage readability in IE by blocking display of both background image and color causing the white h1 to be displayed on white background